recipe/migrations/libparallelproj180.yaml

```
@@ -0,0 +1,8 @@
+__migrator:
+  build_number: 1
+  commit_message: Rebuild for libparallelproj 1.8.0
+  kind: version
+  migration_number: 1
+migrator_ts: 1714455137
+libparallelproj:
+- 1.8.0
```

The `run_exports` is only on `x.x` level and thus we should pin similarly here.
```suggestion
- 1.8
```